helm/event-exporter-app/templates/pss-exceptions.yaml

```diff
@@ -1,28 +0,0 @@
-{{- if .Capabilities.APIVersions.Has "kyverno.io/v2beta1/PolicyException" -}}
```

Comment:
Why is this no longer needed?

Reply:
Because we had exception on stuff I already fixed by making the security context more concrete and secure. The PSS exceptions would ignore `require-run-as-nonroot`, but as I explicitly set `runAsNonRoot: true`, we no longer need to ignore it. And so we can also just remove all exceptions.